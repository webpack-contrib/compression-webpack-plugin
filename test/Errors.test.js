import Plugin from '../src';

describe('Errors', () => {
  test('Validation Error', () => {
    const plugin = () => new Plugin({ minRatio: 'false' });

    expect(plugin).toThrow();
    expect(plugin).toThrowErrorMatchingSnapshot();
  });
});
