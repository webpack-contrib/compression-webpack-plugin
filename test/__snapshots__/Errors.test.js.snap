// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Errors Validation Error 1`] = `
"Validation Error

Compression Plugin Invalid Options

options.minRatio should be number
"
`;
